app/models/project.rb

```diff
@@ -31,6 +31,11 @@ class Project < ActiveRecord::Base
     end
   end
 
+  def self.selected
+    project_names = Team.in_current_season.accepted.pluck(:project_name).uniq
+    Project.in_current_season.where(name: project_names)
```

Comment:
I'm not 100% sure if it's necessary, but I'd have scoped it even more to this:
```ruby
Project.in_current_season.accepted.where(name: project_names)
```
This makes sure, we really only look into the accepted projects (maybe there's a rejected one with the same name...).
What do you think @F3PiX ?